packages/vkui/src/components/Progress/Progress.tsx

```diff
@@ -24,30 +28,57 @@ export interface ProgressProps extends HTMLAttributesWithRootRef<HTMLDivElement>
   /**
    * Стиль отображения прогрессбара
    */
-  appearance?: 'accent' | 'positive' | 'negative';
+  appearance?: LiteralUnion<'accent' | 'positive' | 'negative' | `--${string}`, string>;
```

Comment:
```suggestion
  appearance?: 'accent' | 'positive' | 'negative' | `--${string}` | `#${string}`;
```
с `LiteralUnion` я перегнул, у нас же тут юнион строками 🥲 

предлагаю ограничить HEX'ом, для всего остального, `rgb` / `hsl` /`color()` и т.п., есть `--${string}`

Reply:
Согласен, заменил